src/server/game/Misc/BanManager.cpp

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-GPL2
```

Comment:
the new code should be released under AGPL

Reply:
Okay